packages/cra-template/README.md

```diff
@@ -0,0 +1,43 @@
+# Getting Started with Create React App
+
+This project helps to bootstrap your application with IBM Carbon Design framework. The application
+has 2 sample pages, and those can be accessed over ```/``` and ```/repopage``` respectively.
+It's easy for someone to spin up an app with everything you need to start building with Carbon.
+
```

Comment:
Should this README also include info on how to spin up an application using this template?